pkg/operator/ceph/cluster/controller.go

```diff
@@ -201,6 +201,7 @@ func (c *ClusterController) onK8sNodeAdd(obj interface{}) {
 	newNode, ok := obj.(*v1.Node)
 	if !ok {
 		logger.Warningf("Expected NodeList but handler received %#v", obj)
```

Comment:
Should this be upgraded to an error since we are returning without doing anything? @travisn ?

Reply:
I could go either way, but would lean toward keeping the warning. While it is an unexpected error condition, the admin likely need not be concerned about the error since all this method would do is kick off an orchestration. If the admin sees an error in the log it should be urgent for investigation.

Reply:
Warning sounds good then I think.